community/bolt/src/main/java/org/neo4j/bolt/v2/messaging/Neo4jPackV2.java

```diff
@@ -136,10 +137,10 @@ public void writeLocalTime( long nanoOfDay ) throws IOException
         }
 
         @Override
-        public void writeTime( long nanosOfDayLocal, int offsetSeconds ) throws IOException
+        public void writeTime( long nanosOfDayUTC, int offsetSeconds ) throws IOException
         {
             packStructHeader( 2, TIME );
-            pack( nanosOfDayLocal );
+            pack( nanosOfDayUTC + offsetSeconds * NANOS_PER_SECOND );
```

Comment:
Would be nice to extract this conversion in a helper method because it's repeated in couple places